components/text-callout/Component.jsx

```diff
@@ -80,6 +84,15 @@ export default function TextCallout({
             </p>
           </div>
         )}
+        {buttonText && (
+          <LinkButton
+            buttonText={buttonText}
+            internalUrl={internalLinkUrl}
+            externalUrl={externalUrl}
+            isNewWindow={isNewWindow}
+            className="su-rs-mt-1 su-order-3"
```

Comment:
These additional classes are added to the `LinkButton` component through the use of `cnb` in the component.